python/tests/test_json.py

```diff
@@ -11,6 +11,9 @@
 
 
 @pytest.mark.unit
+@pytest.mark.skip("Can't seem to find the toJson method inside the inference configuration. This might be due to the "
+                  "TextConfig interface addition")
+# TODO: fix this test
```

Comment:
What's the problem here? Are we trying to indirectly use Java reflection or something from Python to find a specific method?

Also, if there was a problem introduced by my PR that added the TextConfig interface, why wouldn't it show up in either my local testing or CI tests? Are python tests not being run as part of the maven build/test?

Reply:
Not sure what's wrong here exactly but the toJson method wasn't found. Also, it could see that the classes weren't initialized correctly with the fromJson method. Maybe it's something specific with pyjnius. 

Also, yes we don't run the python tests as part of the maven tests. CI doesn't have this setup as well. To run it manually, we'll have to install a local anaconda distro and run the `build_jar.py` script in the root as `python build_jar.py --os=<platform>` and then go to `python/tests` and run `pytest .`

Reply:
If we make this part of the maven build/test we'll still have to install a python distro (anaconda preferred). Pinging @sshepel

Reply:
We can decide something here and then make it part of the CI

Reply:
Thanks @ShamsUlAzeem - cc @sshepel 
We definitely need this as part of the maven build, and running as part of the CI, ASAP - we can't have a situation where PRs are (unknowingly, like in the case of my PR) able to break the build even though "mvn test" and CI says they are fine.

https://github.com/KonduitAI/konduit-serving/issues/257
https://github.com/KonduitAI/konduit-serving/issues/258

Reply:
Hm... as for why toJson isn't found - my guess would pyjnius not liking the java 8 default method on the interface?
https://github.com/KonduitAI/konduit-serving/blob/master/konduit-serving-api/src/main/java/ai/konduit/serving/config/TextConfig.java#L30-L46

I've opened an issue so we don't forget: https://github.com/KonduitAI/konduit-serving/issues/259